pkg/nsm/apis/internsm/internsm.proto

```diff
@@ -0,0 +1,105 @@
+syntax = "proto3";
+
+package internsm;
+
+import "github.com/ligato/networkservicemesh/pkg/nsm/apis/common/common.proto";
+
+// InterNsm specifies how Network Service Managers (NSMs) communicate with each other
+// to manage a RemoteConnection between a Network Service Clients (NSC) and Network Service Endpoints (NSE)
+service InterNsm {
+    // RequestRemoteConnection - Service called by one NSM to request a RemoteConnection to a Network Service Endpoint from another NSM
+    rpc RequestRemoteConnection (RemoteConnectionRequest) returns (RemoteConnectionReply);
```

Comment:
By changing  common.RemoteMechanism to `repeated` common.RemoteMechanism in RemoteConnection RmoteConnectionRequest could be eliminated. So all rpc will have the same signature. I think it will unify the processing of message, less number of message less branches to process them.

Reply:
@sbezverk A RemoteConnection has only a single, fully specified RemoteMechanism semantically...

Reply:
Right but slight change we can eliminate an extra message which will simplify processing of message.